rules.txt

```diff
@@ -1,12 +1,395 @@
+---
+##----------------------------
+##    Click-bait Rules
+##----------------------------
+    title+media_title (regex): ["character are you","\\d{1,2} things only","before you die","is this the","you probably d(id|o)n't","are the most","in your life","\\d{1,2} things that","things you probably","reasons you should","probably d(id|o)n't know","\\d{1,2} most important","\\d{1,2} (signs|reasons) (you('re)?|why)","\\d{1,2} most important","d(id|o)n't know about", "(photos|pictures|images) that prove","blow your mind","game of thrones","things that (actually |really )?happen(ed)?","in real life","will blow your","\\d{1,2} dogs who","\\d{1,2} things you", "you should be", "things you d(id|o)n't"]
+    action: report
+    action_reason: "Possible Clickbait - {{match-1}}"
+---
+##----------------------------
+##    Disguised Links Rules, from automoderator library
+##----------------------------
+    moderators_exempt: false
+    body (regex): "(\\[(?P<text>(http|www)\\S+)\\]\\((?!(?P=text))(http|www)\\S+\\))"
+    action: remove
+    action_reason: "Disguised link"
+    modmail: The above {{kind}} by /u/{{author}} was removed because it contained a disguised link. Please investigate immediately as the user was likely trying to circumvent rules.
+---
+##----------------------------
+## Language/Profanity (mostly from automoderator example) - Severe - remove, message user, and modmail
+##----------------------------
+    moderators_exempt: true
+    title+body (regex): ["cock[sz]?([ -]?suck(er[sz]?|ing)?)?","cum(ming|[ -]shot[sz]?)?", "n[ioe0@][g]{1,}[aioe0@]{1,}r[sz]?", "f[aioe0@]{1,}[g]{1,}[aioe03@]{1,}t[sz]?", "k[i1]ke[sz]?"]
+    action: remove
+    action_reason: "Severe Profanity"
+    message: |
+        Your [{{kind}}]({{permalink}}) in /r/{{subreddit}} was automatically removed for use of extreme profanity.
+
+        /r/{{subreddit}} is for productive discussion, so please watch your language.
+    modmail: The above post by /u/{{author}}, contained profanity not becoming of the /r/Linux community. The comment was removed. Please review users account for further action.
+---
+##----------------------------
+## Language/Profanity - (mostly from automoderator example) - report, warn, and modmail
+##----------------------------
+    moderators_exempt: true
+    title+body (regex): ["a(ss|rse|es)([ -]?hole[sz]?)?","b(i|\\*)o?(tch|\\*{3})(y|es)?", "cunt[sz]?","((mother|motha|mutha)[ -]?)?f(u?c?k?k|\\*ck|\\*{0,2}k|\\*{3})(er|ed|ing|s)?", "s(h(i|ar?|\\*)t|\\*{3}|h\\*{2})(s|ter|e|ting)?"]
+    action: report
+    action_reason: "Moderate Profanity"
+    message: |
+        Your [{{kind}}]({{permalink}}) in /r/{{subreddit}} was automatically reported for review for use of profanity.
+
+        /r/{{subreddit}} is for productive discussion, so please watch your language.
+    modmail: The above post by /u/{{author}}, contained profanity not becoming of the /r/Linux community. The comment was removed. Please review users account for further action.
```

Comment:
Remove "The comment was removed"

Reply:
I tested these on my other subreddits and I've now removed the modmail on moderate profanity. Reporting is enough.